### DISCUSSION TITLE
Suspicious events detected by Prisma scan

### DISCUSSION THREAD

**Discussion by Strider221B** [dashingsoft/pyarmor#1766] [Q&A] [answered]
Hi,

We run [prisma scans](https://www.paloaltonetworks.com/prisma/cloud) in our CI and it has flagged the following suspicious activity:

> "135311 wrote a suspicious packed/encrypted binary to /tmp/7lx2gk60/pytransform.so. Packing/encryption can conceal malicious executables. MD5: 5e0b96bec1c90ccc71e720b7d431655b. Command: python ./server.py

Can you please confirm if this is due to code being encrypted/decrypted by pyarmor? Or do we need to investigate this more?

**Comment by jondy** [comment, marked as answer]
I can confirm there is no virus, no undocumented behaviours in `pytransform.so`

**Reply by Strider221B** [reply]
Thanks for confirming!
